### PR TITLE
Scrapper, Liubov Gorbunova - 21FPL2

### PR DESCRIPTION
**Pull request by lubagorbunova**

### CODE REVIEW

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
Not seeing any activity in lab_5_scrapper folder. It is right time to start implementing lab 5.

File: lab_5_scrapper/scrapper.py [hunk 1]
```diff
@@ -90,25 +209,40 @@ def __init__(self, config: Config) -> None:
...
         """
-        pass
+        all_links_bs = article_bs.find_all('a')
+        for link_bs in all_links_bs:
+            link = link_bs.get('href')
+            if link is None:
+                continue
+            elif link[0:8] == 'https://' and 'news' in link and 'from' in link:
+                self.urls.append(link)
+        return ''
```
Comment (dmitry-uraev) [inline lab_5_scrapper/scrapper.py]: is there a more simple method of extracting link?
why you return '' here?

Comment (lubagorbunova) [reply lab_5_scrapper/scrapper.py]: I tried to return link, but then I could extract only one link from a seed url. So I decided to append links to self.url in a loop in order to extract all links from one seed url. And as I append them here, I don't need to return a link. I'm not sure about this return, though.

Comment (lubagorbunova) [reply lab_5_scrapper/scrapper.py]: Done

**Comment by vitazaytseva** [comment]
Well done! You can proceed to work on Lab 6)

**Pull request closed without merging**